### DISCUSSION TITLE
MQTT sur HA

### DISCUSSION THREAD

**Discussion by multinet33** [xlyric/pv-router-esp32#40] [Q&A]
Salut 

est ce que tu sais dire pourquoi les device dans MQTT sous HA passe de valorisé a indisponible puis valorisé et ainsi de suite ?
j'ai cherché deux heures :(
merci

**Comment by xlyric** [comment]
probablement car il doit y avoir qq part des déconnexions mqtt et à la reco
avant la mesure il est en état indisponible.



<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Sans
virus.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Le jeu. 23 nov. 2023 à 18:39, Nicolas ***@***.***> a écrit :

> Salut
>
> est ce que tu sais dire pourquoi les device dans MQTT sous HA passe de
> valorisé a indisponible puis valorisé et ainsi de suite ?
> j'ai cherché deux heures :(
> merci
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/xlyric/pv-router-esp32/discussions/40>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABP6WB66DLC4TPQFZ3QSQ43YF6C5FAVCNFSM6AAAAAA7YDCJASVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVHA4DIMRZGM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Cet e-mail est destiné exclusivement aux personnes dont le nom figure
ci-dessus. Il peut contenir des informations protégées par le secret
professionnel et dont la divulgation est strictement prohibée. Si vous avez
reçu ce message par erreur, nous vous remercions de bien vouloir le
détruire immédiatement et de nous en informer par tous moyens.
( code pénal 226-15 )

**Reply by multinet33** [reply]
A priori pas de reconnexion, je continue mes recherches 
egalement peut tu me preciser pourquoi y'a deux dossier data ? data et data2

**Comment by xlyric** [comment]
comme il faut que j'en vire 1 ;)

Le ven. 24 nov. 2023 à 13:22, Nicolas ***@***.***> a écrit :

> A priori pas de reconnexion, je continue mes recherches
> egalement peut tu me preciser pourquoi y'a deux dossier data ? data et
> data2
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/xlyric/pv-router-esp32/discussions/40#discussioncomment-7660121>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABP6WB2MHLHGQRS6VAB5M6TYGCGPLAVCNFSM6AAAAAA7YDCJASVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TMNRQGEZDC>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Cordialement
Cyril P.
